cypress/features/regression/showEditPodClassic.feature

```diff
@@ -131,16 +131,18 @@ Feature: Show Edit Pod component
       | transparent |
 
   @positive
-  Scenario Outline: Set Show Edit Pod buttonAlign to <buttonAlign>
-    When I select buttonAlign to "<buttonAlign>"
+  Scenario: Set Show Edit Pod buttonAlign to left
+    When I select buttonAlign to "left"
       And I edit Show Edit Pod component
-    Then Show Edit Pod buttons are aligned to <buttonAlign>
-    Examples:
-      | buttonAlign |
-      | right       |
-      | left        |
+    Then Show Edit Pod buttons are aligned to left
 
   @positive
+  Scenario: Set Show Edit Pod buttonAlign to right
```

Comment:
Do we need divide ' show edit pod button align' into two separate 'left and right' test cases?

Reply:
fixed